saleor/settings.py

```diff
@@ -357,16 +357,17 @@ def get_host():
 
 LOGOUT_ON_PASSWORD_CHANGE = False
 
+# SEARCH CONFIGURATION
+DB_SEARCH_ENABLED = True
+PREFER_DB_SEARCH = False  # prefer use of db backend over elasticsearch
```

Comment:
It would be nice if we could have something similar to how Django handles switchable backends for templates engine, emails or static files storage, so a configuration setting that would take path to the proper backend e.g.:
`SEARCH_BACKEND = 'saleor.search.backends.elasticsearch'` or
`SEARCH_BACKEND = 'saleor.search.backends.database'`.